.env.sample

```diff
@@ -46,6 +46,7 @@ REACT_APP_OPENSRP_AUTHORIZATION_URL=https://reveal-stage.smartregister.org/opens
 REACT_APP_OPENSRP_USER_URL=https://reveal-stage.smartregister.org/opensrp/user-details
 REACT_APP_OPENSRP_LOGOUT_URL='https://reveal-stage.smartregister.org/opensrp/logout.do'
 REACT_APP_OPENSRP_OAUTH_STATE=opensrp
+EXPRESS_OAUTH_LOGOUT_URL='http://localhost:3000/logout'
 EXPRESS_OAUTH_GET_STATE_URL=http://localhost:3000/oauth/state
```

Comment:
Lets maintain the naming convention of prefixing `REACT_APP_` to all env vars for the React app

Reply:
ok

Reply:
Yeah they must be prefixed with `REACT_APP_` in order to be picked up by the Create React App scripts right?